jdi-bdd-tests/src/main/java/io/github/com/pages/HtmlElementsPage.java

```diff
@@ -3,42 +3,117 @@
 import com.epam.jdi.light.elements.common.Label;
 import com.epam.jdi.light.elements.complex.Checklist;
 import com.epam.jdi.light.elements.complex.dropdown.Dropdown;
+import com.epam.jdi.light.elements.complex.dropdown.DropdownSelect;
 import com.epam.jdi.light.elements.composite.WebPage;
-import com.epam.jdi.light.elements.interfaces.complex.IsCombobox;
+import com.epam.jdi.light.elements.interfaces.complex.*;
+import com.epam.jdi.light.elements.pageobjects.annotations.NoWait;
+import com.epam.jdi.light.elements.pageobjects.annotations.WaitTimeout;
 import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
+import com.epam.jdi.light.elements.pageobjects.annotations.smart.SName;
+import com.epam.jdi.light.elements.pageobjects.annotations.smart.Smart;
 import com.epam.jdi.light.ui.html.elements.common.*;
-import com.epam.jdi.light.ui.html.elements.complex.MultiSelector;
-import com.epam.jdi.light.ui.html.elements.complex.RadioButtons;
+import com.epam.jdi.light.ui.html.elements.complex.*;
 
 public class HtmlElementsPage extends WebPage {
     //region Simple Elements
-    public static Label jdiTitle;
-    public static Text jdiText;
+
+    @Smart("ui") public static Label jdiTitle;
+    // @UI("[ui=jdi-title]") public static Label jdiTitle;
+    @Smart("ui") public static Text jdiText;
+    // @UI("[ui=jdi-text]") public static Text jdiText;
+
     @UI("[value*='Red Button']") public static Button redButton;
-    public static Button blueButton, dblClickButton, rightClickButton, disabledButton;
+    // @UI("input[type=button].red") public static Button redButton;
+    @UI("[value*='Disabled Button']") public static Button disabledButtonInput;
+    // @UI("input[type=button][disabled]") public static Button disabledButtonInput;
+    public static Button blueButton, dblClickButton, rightClickButton;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use one line for each declaration, it enhances code readability.](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5217753)